src/parsec_security/parsec_threat_model/threat_model.md

```diff
@@ -330,20 +365,22 @@ cleared.
 | 4  | While Parsec is authenticated on the device, anyone can brute force a key ID to execute operations with the key. Unmitigated for the PKCS 11 Provider: all sessions share the login state; if one session logs in then all other opened sessions will also be logged in. Other sessions only need a valid key ID to execute operations with private keys. Mitigated for the TPM Provider: each key is protected by a long, random authentication value, generated by the TPM | All the client's PKCS 11 keys can be used with PKCS 11 operations.                                                         |
 | 5  | *(DSPC)* Parsec does not know about the security quality of a client's Unix password.                                                                                                                                                                                                                                                                                                                                                                                        | All the client's keys can be used, and exported if allowed by the key's policy.                                            |
 | 6  | Parsec does not support using multiple authenticators concurrently ([open issue](https://github.com/parallaxsecond/parsec/issues/272)).                                                                                                                                                                                                                                                                                                                                      | If there is weakness in one authenticator, all the client's keys can be used, and exported if allowed by the key's policy. |
+| 7  | *(JWTS)* The validation steps should parse the JWT-SVID token and make sure the claims of the client's token are the same than the ones returned by the validation operation from the Workload API.                                                                                                                                                                                                                                                                          | All the client's keys can be used, and exported if allowed by the key's policy.                                            |
```

Comment:
> validation steps should parse the JWT-SVID token and make sure the claims of the client's token are the same than the ones returned by the validation operation from the Workload API.

Isn't this something that we should be doing in the service?

Reply:
I believe this validation step should be common to all JWT-SVID token so I think it could be in `rust-spiffe`. This is basically to check that the SPIFFE ID that we are validating is the same as the one that is returned by the validation operation.

Reply:
Hmm, even though we're working on that repo too, I don't think we need to include that here. I'm sure there are lots of cases like this in other libraries we use, for other data flows

Reply:
Ok, yeah, will remove